vllm/model_executor/layers/fused_moe/modular_kernel.py

```diff
@@ -529,6 +530,15 @@ class FusedMoEModularKernel(torch.nn.Module):
     workspace13_buffer = SharedResizableBuffer()
     workspace2_buffer = SharedResizableBuffer()
 
+    class SharedBuffers:
+
+        def __init__(self) -> None:
+            self.fused_out = SharedResizableBuffer()
+            self.workspace13 = SharedResizableBuffer()
+            self.workspace2 = SharedResizableBuffer()
+
+    shared_buffers: list[SharedBuffers] = [SharedBuffers(), SharedBuffers()]
```

Comment:
Why can't we combine shared_buffers with fused_out_buffer = SharedResizableBuffer()? So we don't need to maintain two buffer at the same time.